jmix-flowui/flowui/src/main/java/io/jmix/flowui/view/builder/DetailWindowBuilderProcessor.java

```diff
@@ -105,6 +105,10 @@ protected <E, V extends View<?>> void setupListDataComponent(DetailWindowBuilder
             return;
         }
 
+        if (detailView instanceof StandardDetailView<E> standardDetailView) {
```

Comment:
Probably, it's better moving `setReloadSaved` to `io.jmix.flowui.view.DetailView` interface, to make it more generic